_i18n/nl.yml

```diff
@@ -100,9 +100,6 @@ contribute:
     title: Promoten
     text: Schrijf je pakkende tweets en interessante blogs, of maak je prachtige graphics?
       Help ons AntennaPod wereldberoemd maken.
-  donate: Of, als je niet op een van bovenstaande manieren kunt bijdragen, kun je
-    ook een <a href="https://opencollective.com/antennapod" target="_blank">donatie</a>
-    doen.
```

Comment:
Yeah. Normally we don't (if we just add or remove strings, or change the value of an entry). I think because while building the site it comes across the 'donation' YAML entry and finds content rather than sub-entries and then fails, instead of ignoring it. Will see if I can find out why or get a workaround.

Reply:
I mean, you don't have to spend time on this. As long as the number of translations is still low, it is not a huge issue.